Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+|                 |                     | least-privilege isolation    |
+|                 |                     | between Kubernetes           |
+|                 |                     | workloads, and between       |
+|                 |                     | Kubernetes workloads and     |
+|                 |                     | “external” endpoints running |
+|                 |                     | outside the Kubernetes       |
+|                 |                     | cluster, or running on the   |
+|                 |                     | Kubernetes worker nodes.     |
+|                 |                     | Users should ideally define  |
+|                 |                     | specific allow rules that    |
+|                 |                     | only permit expected         |
+|                 |                     | communication between        |
+|                 |                     | services.                    |
+|                 |                     | |br| |br|                    |
+|                 |                     | Cilium’s network             |
+|                 |                     | connectivity will prevent an |
+|                 |                     | attacker from observing the  |
+|                 |                     | traffic intended for other   |
+|                 |                     | workloads, or sending        |
+|                 |                     | traffic that “spoofs” the    |
+|                 |                     | identity of another pod,     |
+|                 |                     | even if transparent          |
+|                 |                     | encryption is not in use.    |
+|                 |                     | Pods cannot send traffic     |
+|                 |                     | that “spoofs” other pods due |
+|                 |                     | to limits on the use of      |
+|                 |                     | source IPs and limits on     |
+|                 |                     | sending tunneled traffic.    |
++-----------------+---------------------+------------------------------+
+| Observability   | None                | Cilium’s Hubble flow-event   |
+| data            |                     | observability can be used to |
+|                 |                     | provide reliable audit of    |
+|                 |                     | the attacker’s L3/L4 and L7  |
+|                 |                     | network connectivity.        |
++-----------------+---------------------+------------------------------+
+
+.. _bandwidth limitations: https://docs.cilium.io/en/stable/network/kubernetes/bandwidth-manager/
+
+Recommended Controls
+^^^^^^^^^^^^^^^^^^^^
+
+-  Kubernetes workloads should have `defined resource limits`_.
+   This will help in ensuring that Cilium is not starved of resources due to a misbehaving deployment elsewhere in a cluster.
```

Comment:
Does 'elsewhere' have any implications here or can we drop it?

```suggestion
   This will help in ensuring that Cilium is not starved of resources due to a misbehaving deployment in a cluster.
```